config/initializers/saml_idp.rb

```diff
@@ -33,21 +33,6 @@
       email_address: -> (principal) { principal.email }
```

Comment:
The `config.name_id.formats` part? yes, that is used by SamlIdp for both MetadataBuilder and AssertionBuilder.